Assets/Scripts/Game/Game.cs

```diff
@@ -60,28 +60,47 @@ public void PlaceTile(int tileID, int q, int r, int marketIndex = 0)
 		}
 	}
 
-	public void DoubleTile(Tile tile, int q, int r)
+	public void DoubleTile(int tileID, int q, int r)
```

Comment:
You should not need to know what tile is at (q, r) in order to double it. Try making a version of this function that does not require the `tileID` argument.